logstash-core-event-java/src/main/java/com/logstash/Accessors.java

```diff
@@ -134,7 +138,7 @@ private Object fetch(Object target, String key) {
         } else if (target == null) {
             return null;
         } {
-            throw new ClassCastException("expecting List or Map");
+            throw new ClassCastException("expecting List or Map, found " + target.getClass().toString());
```

Comment:
for consistency sake I suggest either putting or not having a comma on these log messages